app/views/shared/_navbar.html.erb

```diff
@@ -7,19 +7,19 @@
     <span class="navbar-toggler-icon"></span>
   </button>
 
-
+<div class="navbar-right">
   <div class="collapse navbar-collapse" id="navbarSupportedContent">
     <ul class="navbar-nav mr-auto">
       <% if user_signed_in? %>
         <% if teacher? %>
           <li class="nav-item active">
-            <%= link_to "Classes", groups_path, class: "nav-link" %>
+            <%= link_to "Classes", groups_path, class: "btn btn-outline-dark" %>
```

Comment:
what? This is the navbar.... why are we putting btn in the navbar? remove btn btn-ouline-dark from all of them, and put class "nav-link" (how it was originally) instead